### ISSUE TITLE
Views + Time Rewards

### ISSUE THREAD

**Issue by MindsGaming** [Joystream/marketing#236]
I think its a great idea to "introduce council paying creators and creators being able to receive this in Atlas in a new section of Atlas Studio." and would love to give my view on rewards. 

Based on views and time 
Content and videos are subjective, what is "better content"?. 

1.) https://play.joystream.org/video/25158
2.) https://play.joystream.org/video/26189

Over subjective rewards for "quality content" let the members decide what is quality content through views and time constraints. 

To do this you will beed some tools:

Light verification
- A way to become verified easily without ID via poldot accounts or using other verification open source services. 

Time Metrics: 
How long did the verified account view the video? 

View Metrics: 
How many verified accounts opened the video to view it? 

Reward Metrics: 

1 min = x $JOY
5 mins = x $JOY
15 mins = x $JOY 
30 mins = x $JOY
1+hour = x $JOY

1 View = x $JOY

**Comment by bedeho** [comment]
There are important technical limits that make this hard presently. We do not do basic viewer accounts, and even if we did, we have nothing close to a real abuse-resistant traffic measurement system. People will easily start to game this if we just use our current naive approach. This is why I suggested using the metrics on the third party platform, so paying for past views - as an indicator of quality, and there we cannot track anything except #views.
